recipients_app/lib/data/models/survey/mapped_survey.dart

```diff
@@ -0,0 +1,22 @@
+import "package:app/data/models/survey/survey.dart";
+import "package:app/data/models/survey/survey_card_status.dart";
+import "package:equatable/equatable.dart";
+
+class MappedSurvey extends Equatable {
+  final Survey survey;
+  final String surveyUrl;
+  final SurveyCardStatus cardStatus;
+  final int? daysToOverdue;
+  final int? daysAfterOverdue;
+
+  MappedSurvey({
```

Comment:
this can be const

Reply:
fixed [db61371](https://github.com/socialincome-san/public/pull/413/commits/db613717cad931f1ff6cdb0df3dbdfe387ec6bb0)